packages/api/test/utils/utils.ts

```diff
@@ -14,15 +14,20 @@ export function getTestServer(): {baseUrl: string; server: FastifyInstance} {
 
   server.addHook("onError", (request, reply, error, done) => {
     // eslint-disable-next-line no-console
-    console.log(error);
+    console.log(`onError: ${error}`);
     done();
   });
 
   before("start server", async () => {
     await new Promise((resolve, reject) => {
       server.listen(port, function (err, address) {
-        if (err !== null && err != undefined) reject(err);
-        else resolve(address);
+        if (err !== null && err != undefined) {
+          reject(err);
+          console.log("error on listen", err);
+        } else {
+          console.log("listen on", address);
+          resolve(address);
+        }
```

Comment:
Is this change necessary?

Reply:
It was handy during debugging, and thought to leave it as it does not lead to a flood of log entries.

Reply:
Removed!